conda-forge.yml

```
@@ -6,13 +6,13 @@ build_platform:
   osx_arm64: osx_64
 conda_build:
   pkg_format: '2'
+conda_build_tool: "conda-build"
+conda_install_tool: "conda"
+conda_solver: "libmamba"
 conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
 provider:
   linux_aarch64: default
   linux_ppc64le: default
```

```suggestion
  linux_ppc64le: default
  remote_ci_setup:
    - conda-forge-ci-setup=3
    - conda>=23.9
    - conda-libmamba-solver>=23.9
    - conda-build>=3.27
```

Maybe a gentle nudge here will help.